src/intent.html

```diff
@@ -245,9 +247,19 @@ <h3 id="mixing_intent_dictionaries">Intent Concept Dictionaries</h3>
      <a data-cite="INFRA#ascii-case-insensitive">ASCII case-insensitive</a>
      and also normalize
      <q>`_`</q> (U+00F5) and <q>`.`</q>  (U+002E) to <q>`-`</q> (U+002D).
-     If the speech hints are not being used
-     and the concept name is being read then each of  `-`, `_` and `.` should be
-     read as an inter-word space.</p>
+     An intent matches an entry in a concept list when the normalized name, the fixity property
+     (which may be defaulted in the concept dictionary), and the number of arguments all match.
+     If a match is found, the speech hint in the list
+     should be used be used as a guide for the generation of suitable text
+     for the <a href="#intent_known_concept">known concept</a>.
+     If the intent does not match any entries, then it is treated as
```

Comment:
```suggestion
     Recall that any concept not currently known to the AT is an  <a href="#intent_known_concept">known concept</a> and treated as a <a href="#intent_literal">literal</a>.
```

Reply:
@NSoiffer  I suggest you click the "commit suggestion" button above and call it done:-) but I just saw your last comment which crossed with me making the above, feel free to further adjust, but it's late here and I'm mostly offline tomorrow so don't wait for a review from me.